src/framework/mlt_producer.h

```diff
@@ -80,6 +80,25 @@ struct mlt_producer_s
 	 */
 	int ( *get_frame )( mlt_producer, mlt_frame_ptr, int );
 
+	/** Seek to s specified position (virtual function).
+	 *
+	 * \param mlt_producer a producer
+	 * \param mlt_position a frame pointer by reference
+	 * \param position set the "play head" position of the producer
+	 * \return false
+	 */
+	int ( *seek )( mlt_producer, mlt_position );
```

Comment:
Since this adds new members to a public struct, it does break ABI. That is OK for now but FYI. Hold off on the symbol version updates. Looks like this might be 7_0_0. In new major version, we can change some things to a private. But also keep in mind that you want to avoid breaking ABI a good while after upgrading it.